code/game/machinery/pipe/construction.dm

```diff
@@ -147,14 +147,16 @@ Buildable meters
 /obj/item/pipe/attack_self(mob/user)
 	return rotate()
 
-/obj/item/pipe/proc/get_pipe_cache(type, direction)
-	var/static/list/obj/machinery/atmospherics/check_cache
+/proc/get_pipe_construction_cache(type, direction)

```

Comment:
Why remove the specification?

Reply:
this does not need to go a global proc no